platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/eventhandling/ConsensusRoundHandlerTests.java

```diff
@@ -114,7 +115,8 @@ void normalOperation() throws InterruptedException {
         final long consensusRoundNumber = 5L;
         final ConsensusRound consensusRound = mockConsensusRound(keystoneEvent, events, consensusRoundNumber);
 
-        consensusRoundHandler.handleConsensusRound(consensusRound);
+        final StateAndRound handlerOutput = consensusRoundHandler.handleConsensusRound(consensusRound);
+        assertNotEquals(null, handlerOutput, "new state should have been created");
```

Comment:
I think state reservation assertions should also be made in this test.

Reply:
Added assertions to verify that the returned state has a reservation